### DISCUSSION TITLE
curso:  Fundamentos do ASP.NET 6, módulo: ASP.Net, aula: cache

### DISCUSSION THREAD

**Discussion by EddyLopes** [balta-io/forum#517] [Tira Dúvidas]
gostaria de saber como faz o cache quando a consulta é paginada?

**Comment by andrebaltieri** [comment]
Bom dia, @EddyLopes, como vai?

Você pode ler uma quantidade de informação, cachear e paginar os pedaços deste cache:
https://balta.io/blog/aspnet-memory-cache
